app/views/inventories/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="container mt-4">
+  <h1 class= "mb-5"><%= link_to "Inventories", branch_inventories_path(@branch_id), class: "text-decoration-none" %> / New Inventory</h1>
```

Comment:
It is giving errors 
- when we aren't selecting any ingredient.
- when new ingredient is selected but quantity not selected
<img width="1001" alt="image" src="https://github.com/rishabh-vinsol/restaurant-pos/assets/64541606/347fed8e-3ace-4b41-a171-7a41797bdbd5">

Reply:
Fixed